packages/tokens/scripts/extract-carbon-parts/convertObjectToDtcgFormat.ts

```diff
@@ -0,0 +1,209 @@
+          return {
+            '$type': 'color',
+            '$value': value,
+            'group': group || 'cds-generic-color',
+            'private': true,
+            'cds-original-value': value
+          };
+        case 'cubic-bezier':
+          return {
+            // TODO convert to `cubicBezier` per DTCG specifications when we are sure that Style Dictionary process it correctly
+            // see: https://www.designtokens.org/tr/drafts/format/#cubic-bezier
+            '$type': 'cubic-bezier',
+            '$value': value,
+            'group': group || 'cds-generic-cubic-bezier',
+            'private': true,
+            'cds-original-value': value
+          };
+        case 'duration':
+          const convertedDurationValue = convertDurationValue(value);
+          if (convertedDurationValue !== undefined) {
+            return {
+              '$type': 'duration',
+              '$value': convertedDurationValue.$value,
+              'unit': convertedDurationValue.unit,
+              'group': group || 'cds-generic-duration',
+              'private': true,
+              'cds-original-value': value
+            };
+          } else {
+            const unknownDurationToken = returnUnknownToken(value, `🚨 convertDurationValue: value for key "${key}" / group "${group}" is not in the expected format:`);
+            return unknownDurationToken;
+          }
+        case 'font-family':
+          return {
+            '$type': 'font-family',
+            '$value': value,
+            'group': group || 'cds-generic-font-family',
+            'private': true,
+            'cds-original-value': value
+          };
+        case 'font-size':
+          const convertedFontSizeValue = convertSizeValue(value, false);
+          if (convertedFontSizeValue !== undefined) {
+            return {
+              '$type': 'font-size',
+              '$value': convertedFontSizeValue.$value,
+              'unit': convertedFontSizeValue.unit,
+              'group': group || 'cds-generic-font-size',
+              'private': true,
+              'cds-original-value': value
+            };
+          } else {
+            const unknownSizeToken = returnUnknownToken(value, `🚨 convertSizeValue: value for key "${key}" / group "${group}" is not in the expected format:`);
+            return unknownSizeToken;
+          }
+        case 'font-weight':
+          return {
+            '$type': 'font-weight',
+            '$value': value,
+            'group': group || 'cds-generic-font-weight',
+            'private': true,
+            'cds-original-value': value
+          };
+        case 'line-height':
+          return {
+            '$type': 'number',
+            '$value': value,
+            'group': group || 'cds-generic-line-height',
+            'private': true,
+            'cds-original-value': value
+          };
+        case 'letter-spacing':
+          const convertedLetterSpacingValue = convertSizeValue(value, false);
```

Comment:
```suggestion
          const convertedLetterSpacingValue = convertSizeValue(value);
```
[Issue] By passing in false the outputted values are in rems not px, but the unit is still px. This is causing incorrect letter-spacing styles to be generated.

Example: `carbon-extracted/typography/styles.json`
```
"letter-spacing": {
  "$type": "letter-spacing",
  "$value": 0.16,
  "unit": "px",
  "group": "cds-typography",
  "private": true,
  "cds-original-value": "0.16px"
}
```

Reply:
My mistake, I see now in the original value this is as intended.